src/main/java/com/google/sps/servlets/RetrieveActivitiesServlet.java

```diff
@@ -0,0 +1,105 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.google.gson.Gson;
+
+import com.google.sps.data.Activity;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+
+/** 
+* Servlet that retrieves actvities of a specified category.
+*/
+@WebServlet("/retrieve-activities")
+public class RetrieveActivitiesServlet extends HttpServlet {
+  private static final String GAMES_CATEGORY = "games";
```

Comment:
These could be changed to enums or ints

Reply:
Also change these to public static final and move them Activity.java

Reply:
Fixed it.